guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+
+1. Open the folders of the skins you want to mix from in your file explorer. These folders can be found by navigating to the `Skins\` directory of your osu! installation. By default this will be located at `C:\Users\[User Name]\AppData\Local\osu!\Skins\`. You can locate this by pressing `CTRL + O` in the osu! window to bring up settings, clicking `Open osu! folder`, and navigating to the `Skins\` directory. Alternatively, you can go to settings, navigate to the `SKIN` section, and click `Open current skin folder` for all skins you want to mix.
+2. Create a new folder in the `Skins\` directory and name it something that you will remember, we will call our example `mixedskin`.
+3. Decide what images from which skins you want to use. For example, we want to use the cursor from `skin1`, the hitcircle and hitcircle overlay from `skin2`, and so on, into `mixedskin`.
+4. Move these images into the folder we created. We move `cursor.png` and `cursortrail.png` from `skin1`, and `hitcircle.png` and `hitcircleoverlay.png` from `skin2`, and all of their HD equivalents, if present, into `mixedskin`
+5. Reload your skins with `CTRL + ALT + SHIFT + S` , and select the mix skin from the drop down list. With our example, it should be the default skin with `skin1`'s cursor and `skin2`'s hitcircle. Make sure to reload your skin after all changes you make for those changes to take effect.
+6. Repeat steps 3 and 4 until you have reached your desired result.
+
+If you are mixing `default-x.png`, `score-x.png`, and/or `combo-x.png` numbers, please also read the `The skin.ini` section of this guide.
+
+## Mixing animations
+
+The steps for mixing animations are largely similar to the steps for mixing images, with only a few major differences:
+
+- You will need to delete **all** animations and/or images of the same element already present in your mixed skin. (e.g. delete all `followpoint-x.png` files present if you wish to use followpoints from a different skin).
+- You will have to transfer **all** animation files.
+- You might need to make sure that the `AnimationFramerate:` command in your mixed skin's `skin.ini` (talked about in the `The skin.ini` section of this guide) is set to the same value as it was set in the skin you are taking the animation from for some animations to look as they should. **Note that this will probably break other animations, such as `followpoint-x.png`, so most of the time it is better not to specify this command at all.**
+
+Apart from these differences above, the process is the same as described in the `Mixing images` section of this guide.
+
+## Mixing sounds
+
+The steps for mixing sounds are also largely similar to the `Mixing images` section of this guide, with the difference that the files you are replacing will be .mp3 or .wav instead of .png. Follow the instructions in the `Mixing images` section, using sound files instead.

```

Comment:
probably want to point out that there are a lot of files, but usually you can get a full menu sound set with like 7 sounds or so.